GliaWidgets/Public/Glia/Glia+StartEngagement.swift

```diff
@@ -141,7 +141,8 @@ extension Glia {
                 getSecureUnreadMessageCount: environment.coreSdk.getSecureUnreadMessageCount,
                 messagesWithUnreadCountLoaderScheduler: environment.messagesWithUnreadCountLoaderScheduler,
                 secureMarkMessagesAsRead: environment.coreSdk.secureMarkMessagesAsRead,
-                downloadSecureFile: environment.coreSdk.downloadSecureFile
+                downloadSecureFile: environment.coreSdk.downloadSecureFile,
+                isAuthenticated: { [environment] in (try? environment.coreSdk.authentication(.forbiddenDuringEngagement).isAuthenticated) ?? false }
```

Comment:
perhaps it makes sense to wrap it inside do/catch and log error message for debug purposes?

Reply:
@yurii-glia  But what the benefit do we get from those logs in this particular case? Visitor can be unauthenticated, and of course  this will throw error, but it doesn't seem important for current case, because the only thing we do care here is if authentication happened or not. In my opinion it will unnecessarily pollute console.

Reply:
Added d0a2ccb8459afbd7010348b54f8da86f9fec6a52.